domain-cc/cc-app/src/python_src/util/sanitizer.py

```diff
@@ -0,0 +1,6 @@
+def sanitize(obj):
+    if isinstance(obj, bool):
+        return bool(str(obj).replace("\r\n", "").replace("\n", ""))
+    if isinstance(obj, int):
+        return int(str(obj).replace("\r\n", "").replace("\n", ""))
+    return str(obj).replace("\r\n", "").replace("\n", "")
```

Comment:
This function seems to make this type of [log injection warning](https://github.com/department-of-veterans-affairs/abd-vro/security/code-scanning/1453) go away. Just noting here that we're not actually stripping PII. `sanitize()` does make PII `str`'s safe to log freely into datadog